spec/spec_helper.rb

```diff
@@ -8,12 +8,12 @@
 require 'readyset'
 require_relative 'shared_examples'
 
+# This is a bit of a hack. Combustion doesn't appear to support migrating multiple databases, so we
```

Comment:
Was moving this comment intentional? I think it should prob stay above the file copying lines since that's the code it refers to

Reply:
Just bumping this to make sure you saw it 😄

Reply:
Sorry! Fixed now.